Components/badge.css

```diff
@@ -0,0 +1,177 @@
+:root{
+    --status-online-color : yellowgreen;
+    --status-away-color : yellow;
+    --status-busy-color : red;
+    --badge-blue-bg : blue;
+    --badge-pink-bg : hotpink;
+    --badge-red-bg : red;
+    --badge-text-color : white;
+}
+
+/* Badge on avatar*/
+.badge-on-avatar-xl{
+    height: 200px;
+    width: 200px;
+    position: relative;
+}
+
+.badge-on-avatar-l{
+    height: 150px;
+    width: 150px;
+    position: relative;
+}
+
+.badge-on-avatar-m{
+    height: 100px;
+    width: 100px;
```

Comment:
Try to use `rem` unit for this. It will help in making images responsive

Reply:
Sure will covert it to rem. Thanks for your review :)